patches/tools-crates-gnrt-gen.rs.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tools/crates/gnrt/gen.rs b/tools/crates/gnrt/gen.rs
```

Comment:
at least some of these patches can go to upstream, we'll figure out the details in a follow-up